statics/src/main/java/com/codeborne/selenide/Selenide.java

```diff
@@ -582,10 +582,22 @@ public static ElementsCollection getElements(By criteria) {
     return getSelenideDriver().findAll(criteria);
   }
 
+  /**
+   * @see <a href="https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/JavascriptExecutor.html
```

Comment:
A proper way to refer other Java classes in javadoc is mentioning class name + method name with argument types:

```java
/**
 * @see org.openqa.selenium.By#className(java.lang.String)
 */
```

This way we don't need to bother about long or changing URLs.

Reply:
true,  but then you would need to find out the page in google, the idea is to provide a link still to help the user.

I suggest to add an ignore pattern in the checkstyle related rule in my next commit, that allow longer line for " * @see " which is then more url friendly. That also permit to include a longer text inside the link itself.

Reply:
@tyge68 No, you don't need to google. IDEA can automatically download sources jar from maven central repo. You can easily browse sources/javadoc of any open-sources projects right in IDEA (not sure about Eclipse/NetBeans).

Reply:
+